src/components/NavDropdown.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <el-dropdown trigger="click" class="p-2">
+      <span class="el-dropdown-link flex items-center">
+        <img class="h-5 rounded-full bg-black mr-1" src="https://t4.ftcdn.net/jpg/03/46/93/61/360_F_346936114_RaxE6OQogebgAWTalE1myseY1Hbb5qPM.jpg" alt="Avatar">My User Name<i class="el-icon-arrow-down el-icon--right"></i>
+      </span>
+    <el-dropdown-menu slot="dropdown">
+      <el-dropdown-item><router-link to="/">Profile</router-link></el-dropdown-item>
```

Comment:
Do not put the router-link here, the route should be changed by JS. Use the 
`this.$router.push('/login')`

You can use the command event that is on the el-dropdown, to handle the changes

Reply:
I have used @click.native. However, I get a NavigationDuplicated error when going to the same route two times in a row:
<img width="1440" alt="Screenshot 2021-08-20 at 09 29 07" src="https://user-images.githubusercontent.com/89005333/130190049-41d83c1c-745a-4711-a511-3085a633558b.png">

How should I handle this?

Reply:
I thought about pushing the new route only if it's not the same as the current